src/modules/Footer/Social/Social.tsx

```diff
@@ -9,10 +9,8 @@ bibendum.`;
 const Social = () => {
   return (
     <Container>
```

Comment:
חיים שלי את!
ראיתי שכתבת לעצמך הערה עם trade off למטה ❤️
איך היא אוהבת טריידאופים...

please remove all the unused stuff from this file (and others)

Reply:
I like the most this conversation :) hearts and emojis